frontend/src/history.ts

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from 'history'
+
+export const history = createBrowserHistory()
```

Comment:
do we use it?

Reply:
We use library `connected-react-router` for history.push action making through React Router 5 after successful login operation, and its a demand of this lib that this operation should be in a separate file. I have PA-56 task for refactoring this process, so lets leave at as it is by now.